skimage/measure/_regionprops.py

```diff
@@ -603,11 +854,7 @@ def __iter__(self):
         return iter(sorted(props))
 
     def __getitem__(self, key):
-        value = getattr(self, key, None)
-        if value is not None:
-            return value
-        else:  # backwards compatibility
-            return getattr(self, PROPS[key])
```

Comment:
I removed this if/else case since all old properties now have attributes via the `DeprecatedProperties` mixin class.

Reply:
@grlee77 perhaps we can keep this and just add the old property names to the dict?

Reply:
Oh right but that doesn't help with attribute access. Perhaps we can add a `__getattr__` to the class?